browser/test.html

```diff
@@ -0,0 +1,19 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <title>Flickr api test</title>
+    <link rel="stylesheet" type="text/css" href="style.css">
```

Comment:
HTML5 has default types, so `link` doesn't need a `type` attribute for CSS, it is assumed to be CSS if the relation is "stylesheet"